tests/Unit/Domain/CoordinateMaps/Test_DiscreteRotation.cpp

```diff
@@ -21,81 +21,99 @@
 
 SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.DiscreteRotation.Specific1D",
```

Comment:
Here too. Instead of sections factor into functions that are called from the SPECTRE_TEST_CASE